packages/editor/src/client/locales/de.json

```diff
@@ -11,13 +11,13 @@
         "metadata": "Metadaten",
         "noSlug": "Artikel nicht publiziert. Bitte Slug erfassen.",
         "notFound": "Artikel nicht gefunden",
-        "pending": "Publikation des Artikels geplant am {{date, lll}}",
+        "pending": "Publikation des Artikels geplant am {{date, ccc PP p}}",
```

Comment:
can you explain what's c P p mean?

Reply:
c formats the day (ccc -> Mon), P the date (12/12/2021) and p the time (p for 00:00 and pp for 00:00:00). You can read the documentation [here](https://date-fns.org/v2.23.0/docs/format)